extensions/mssql/src/contextProvider.ts

```diff
@@ -40,6 +41,7 @@ export default class ContextProvider {
 		let iscloud: boolean;
 		let edition: number;
 		let isCluster: boolean = false;
+		let serverVersion: string;
```

Comment:
I'm curious if there's anything else that can be used instead of string. How does the comparison in the when clause work, out of curiosity?

Reply:
One thing just popped into my head. If we just take the server's major version, we can use a number here, and then comparisons are much easier. Something to think about.